docs/src/main/sphinx/connector/influxdb.rst

```diff
@@ -0,0 +1,197 @@
+=======================
+InfluxDB connector
+=======================
+
+The InfluxDB Connector allows access to `InfluxDB <https://www.influxdata.com/>`_ data from Trino.
+This document describes how to setup the InfluxDB Connector to run SQL queries against InfluxDB.
+
+Requirements
+------------
+
+To connect to InfluxDB, you need:
+
+* InfluxDB 1.x or higher.
+* Network access from the Trino coordinator and workers to InfluxDB.
+
+Configuration
+-------------
+
+To configure the InfluxDB connector, create a catalog properties file
+``etc/catalog/influxdb.properties`` with the following contents,
+replacing the properties as appropriate:
+
+.. code-block:: properties
+
+    connector.name=influxdb
+    influx.host=localhost
+    influx.port=8086
+    influx.protocol=http
+    influx.username=username
+    influx.password=password
+
+Configuration properties
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. list-table:: The following configuration properties are available
+    :widths: 35, 55, 10
+    :header-rows: 1
+
+    * - Property name
+      - Description
+      - Default
+    * - ``influx.host``
+      - Hostname of the InfluxDB server to connect to. This property is required.
+      -
+    * - ``influx.port``
+      - Port of the InfluxDB server to connect to. This property is required.
+      -
+    * - ``influx.protocol``
+      - Protocol of the InfluxDB server to connect to.
+      - ``http``
+    * - ``influx.username``
+      - User name to use to connect to InfluxDB.
+      -
+    * - ``influx.password``
+      - Password name to use to connect to InfluxDB.
+      -
+    * - ``influx.cacheTimeOut``
+      - How long for meta data (table schemas、columns schemas) to cache in memory.
+      - ``Never``
+    * - ``influx.connectTimeOut``
+      - The socket connect timeout to InfluxDB server.
+      - ``10s``
+    * - ``influx.writeTimeOut``
+      - The socket write timeout to InfluxDB server.
+      - ``10s``
+    * - ``influx.readTimeOut``
+      - The socket read timeout to InfluxDB server.
+      - ``10s``
+    * - ``influx.optimize.filterPushDownEnable``
+      - Set true to enable filter pushdown.
+      - ``false``
+    * - ``influx.optimize.limitPushDownEnable``
+      - Set true to enable limit pushdown.
+      - ``false``
+    * - ``influx.optimize.projectionPushDownEnable``
+      - Set true to enable projection pushdown.
+      - ``false``
+    * - ``influx.aggregationPushDownEnable``
+      - Set true to enable aggregation pushdown.
+      - ``false``
```

Comment:
These properties should be revised when addressing comments. See other comments in config class.